src/prpy/planning/retimer.py

```diff
@@ -154,6 +154,15 @@ def __init__(self, do_blend=True, do_shortcut=True, blend_radius=0.5,
             'time_limit': float(timelimit),
         })
 
+    @PlanningMethod
+    def RetimeTrajectory(self, robot, path, options=None, **kw_args):
+        from copy import deepcopy
+        if options is None:
+            options = {}
+        new_options = deepcopy(options)
+        new_options['time_limit'] = kw_args.get('timelimit', 3.)
+        return super(HauserParabolicSmoother, self).RetimeTrajectory(
+            robot, path, options=new_options, **kw_args)
```

Comment:
Um.. we did also write `or_parabolicsmoother`.  Perhaps we should just change it to be `timelimit` there?

https://github.com/personalrobotics/or_parabolicsmoother/blob/master/include/or_parabolicsmoother/ParabolicSmootherParameters.h#L117